LoopKit.xcodeproj/project.pbxproj

```diff
@@ -4185,6 +4186,7 @@
 				A9498D7F23386C3300DAA9B9 /* GlucoseThreshold.swift in Sources */,
 				A9E675D522713F4700E25293 /* WalshInsulinModel.swift in Sources */,
 				A9E675D622713F4700E25293 /* GlucoseEffectVelocity.swift in Sources */,
+				B40C43912707408400F5D86C /* DeliveryLimits.swift in Sources */,
```

Comment:
I don't see this new file in the PR?

Reply:
It was already there in `LoopKit`, but now also available in `LoopKit-watchOS`.